Improve wiki pages (#31)
LambdAurora/AurorasDecorations: wiki/amethyst_lantern.md

<!--description:Learn everything about the Amethyst Lantern, a very cute and practical lantern.-->
<!--thumbnail:images/amethyst_lantern.png-->

> Amethyst Lantern is a type of lantern which primary color is purple, like amethyst! 

Reviewer: ```suggestion
The Amethyst Lantern is a type of lantern which primary color is purple, like amethyst! 
```

They emit some purple particles while placed.

> In addition to the cosmetic part of this lantern, the Amethyst Lantern will cancel natural mob spawning in a 32 blocks radius in all directions around it. 

Reviewer: ```suggestion
In addition to its cosmetic function, the Amethyst Lantern will cancel natural mob spawning in a 32 block radius in all directions around it. 
```
